soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3947,14 +3945,18 @@ void DrawRandoEditor(bool& open) {
                 ImGui::PushItemWidth(-FLT_MIN);
 
                 //Starting Age
-                //Disabled when Forest is set to Closed
-                bool disableRandoStartingAge = !CVar_GetS32("gRandomizeForest", 0);
-                const char* disableRandoStartingAgeText = "This option is disabled because \"Forest\" is set to \"Closed\".";
+                //Disabled when Forest is set to Closed or under very specific conditions
+                //RANDOTODO: Replace magic number checks with enums
+                bool disableRandoStartingAge = (CVar_GetS32("gRandomizeForest", 0) == 0) || //If Forest is closed...
+                    (CVar_GetS32("gRandomizeDoorOfTime", 0) == 0) && //or Door of Time is Closed
+                    (CVar_GetS32("gRandomizeShuffleOcarinas", 0) == 0) && //and ocarinas are not shuffled
+                    (CVar_GetS32("gRandomizeLogicRules", 0) == 0); // and logic is glitchless
```

Comment:
```suggestion
                bool disableRandoStartingAge =  (CVar_GetS32("gRandomizeLogicRules", 0) == 0) && // glitchless logic
                ((CVar_GetS32("gRandomizeForest", 0) == 0) || // Closed Forest
                 ((CVar_GetS32("gRandomizeDoorOfTime", 0) == 0) && // Closed Door of Time 
                 (CVar_GetS32("gRandomizeShuffleOcarinas", 0) == 0)));  // ocarinas not shuffled
                    
```

the way it was before closed forest would make it impossible to randomize age even when generating a no logic seed